tools/cli/src/commands/interactive/session.ts

```
@@ -0,0 +1,362 @@
+  /**
+   * Unsubscribe from session events
+   */
+  off(event: string, handler: (data?: any) => void): void {
+    this.eventListeners.get(event)?.delete(handler);
+  }
+
+  /**
+   * Emit a session event
+   */
+  private emit(event: string, data?: any): void {
+    this.eventListeners.get(event)?.forEach((handler) => handler(data));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)